src/components/ui/Panel.jsx

```diff
@@ -311,7 +314,10 @@ class Panel extends React.Component {
             }}
             {...(isMobile && resizable && this.getEventHandlers())}
           >
-            {floatingItems && <FloatingItems items={floatingItems} />}
+            {floatingItemsLeft && size !== 'maximized' &&
```

Comment:
Why do we need the `size !== 'maximized'` test when it wasn't used for the right items?

Reply:
![image](https://user-images.githubusercontent.com/2981774/109991620-c39e2b80-7d0a-11eb-962f-05cafab278c8.png)

I added this check in order to avoid this situation. This should not happen currently as we have no button visible in maximized state.

Maybe we should add this check for right floatingItems ?

Reply:
> Maybe we should add this check for right floatingItems ?

Yes probably :+1: